relyance.yaml

```
@@ -0,0 +1,22 @@
+# Relyance Compliance Inspector Configuration
+
+# TODO: Replace text in < > with actual information from your environment
+
+# Product and Repository Metadata
+# Purpose: Useful to slice/dice the Data Inventory and Data Map in Relyance
+repository: simulator
+
+# Services and Code Relationship
+# Purpose: Configuration used to visualize the code base and relationships between microservices and code.
+# Note: Directory paths are relative to the repository's root directory. 
+# Note: Need a separate entry for each microservice. See example below.
+
+services:
+  - service: simulator
+    product: World App
```

```suggestion
    product: World ID
```
Our team name should be world ID?